src/Components/CreateImageWizard/CreateImageWizard.scss

```diff
@@ -100,19 +94,13 @@ div.pf-v5-c-alert.pf-m-inline.pf-m-plain.pf-m-warning {
     Temporarily commenting out this code, until we figure out how the handle
```

Comment:
Looks like we are re adding the flex, so this comment and the above TO-DO doesn't make sense any more. Should we remove it?

Reply:
Sure, No more confusing TODOs! 😄

Reply:
If we need any followup in the future, let's just create a ticket right now, so that we do not forget though! :)